bot/bot.py

```diff
@@ -170,6 +174,16 @@ def clear(self) -> None:
 
     async def close(self) -> None:
         """Close the Discord connection and the aiohttp session, connector, statsd client, and resolver."""
+        # Done before super().close() to allow tasks finish before the HTTP session closes.
+        with suppress(Exception):
+            [self.unload_extension(ext) for ext in tuple(self.extensions)]
+            [self.remove_cog(cog) for cog in tuple(self.cogs)]
```

Comment:
`suppress()` will suppress the first exception is encounters and then resume execution on the line following the end of the with statement (i.e. line 183). This effectively prematurely ends the unloading/removal operation. 

Each call needs to be surrounded in a try-except.  Basically, it needs to be similar to how discord.py does it, which you used to follow but changed for some reason.